### DISCUSSION TITLE
How to integrate a work calendar (G cal) where admin has blocked ability to share it

### DISCUSSION THREAD

**Discussion by jck2** [ivan-lednev/obsidian-day-planner#494] [Q&A] [answered]
Apologies if this is covered in the README but I have a calendar setup through my work email that uses google calendar. I was able to figure out a way to view it via my personal google calendar (just the event times/busy status), but that method doesn't seem to allow sharing the iCS link to 3rd party apps. 

Is there any known workaround for this issue?

**Comment by ivan-lednev** [comment, marked as answer]
Hello! If your org admin has prohibited this, then you probably should not do it, otherwise you might get yourself into trouble.

**Reply by jck2** [reply]
Fair enough, thanks for the quick reply
